sensu-plugins-load-checks.gemspec

```diff
@@ -25,7 +25,7 @@ Gem::Specification.new do |s|
   s.platform               = Gem::Platform::RUBY
   s.post_install_message   = 'You can use the embedded Ruby by setting EMBEDDED_RUBY=true in /etc/default/sensu'
   s.require_paths          = ['lib']
-  s.required_ruby_version  = '>= 2.0.0'
+  s.required_ruby_version  = '>= 2.1.0'
```

Comment:
makes it a breaking change (which is OK as long as it called out appropriately in CHANGELOG and versioned.